examples/language-modeling/run_mlm_flax.py

```diff
@@ -566,6 +605,8 @@ def tokenize_function(examples):
     ).create(model.params)
 
     # Create learning rate scheduler
+    if training_args.warmup_steps == 0:
+        training_args.warmup_steps = 1
```

Comment:
No, we don't change training arguments without at least warning the user (and we really shouldn't change them at all).

Reply:
I'll move this to the other PR - but warmup_steps = 1 gives the same results as no warmup (which is expected with warmup_steps = 0) and warmup_steps = 0 causes the Flax optimizer to return NaNs. Should we keep this and warn the user ?

Reply:
I'm fine with a warning if there is a problem with the 0 value (the warning should explain why we do the change).